### ISSUE TITLE
about the 'NAN’ loss during training

### ISSUE THREAD

**Issue by XiejcSYSU** [xiangwang1223/neural_graph_collaborative_filtering#43]
Hi！why the first version of loss function will cause 'nan' loss during training? Thank you.

**Comment by Zhaominghao1314** [comment]
'tf.log' resuts in 'inf',  just change to 'maxi = tf.log(tf.clip_by_value(tf.nn.sigmoid(pos_scores - neg_scores),1e-8, 1))' in  L284 of NDCG.py
